mne/morph.py

```diff
@@ -363,7 +363,7 @@ def apply(self, stc_from, output='stc', mri_resolution=False,
         if not isinstance(output, str):
             raise TypeError('output must be str, got type %s (%s)'
                             % (type(output), output))
-        out = _apply_morph_data(self, stc)
+        out = _apply_morph_data(self, stc, mri_resolution=True, mri_space=True)
```

Comment:
@TommyClausner this is the part that seems like a bug to me. When `morph.apply`'ing to a STC, effectively in the code you were always setting `mri_resolution=True, mri_space=True` (see below; here I've done it explicitly just to keep the functionality equivalent). Was this intentional? It seems like it ignores the arguments to `morph.apply`.